src/test/java/jsUtils.js

```diff
@@ -165,3 +165,24 @@ function generateUUID(){
     var uuid = '' + java.util.UUID.randomUUID();
     return uuid
 }
+
+/** To validate expected result and actual result with dynamic value */
```

Comment:
Also define the details of parameters. what is response? and what is constant? and what this function will return if validation is success